testdata/p4_14_samples_outputs/01-BigMatch-first.p4

```diff
@@ -24,14 +24,14 @@ header vag_t {
 }
 
 struct metadata {
-    @name("ing_metadata") 
+    @name(".ing_metadata") 
```

Comment:
There is a globalname annotation in the spec for this purpose. Are we planning to use it?

Reply:
We could certainly switch over to it, but I'd prefer to do it in a separate PR. This just maintains the same mechanism we've been using up to now, but applied in more cases.